Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,66 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-593-gc17fa86-src.tar.gz"
+  version "1.6.7-593"
+  sha256 "21579278c732674149bfe55a4f408c47f44f7d2df6bfc71f3b12f004aa14bd1b"
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
```

Comment:
Indeed, and a couple other places too I think. I'd like to stamp it out though; one of our long-term goals has been to bottle `devel` and we can't do that if it's expected to move around.